boards/pulga/Kconfig

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "nrf52840dk" if BOARD_NRF52840DK
+
+config BOARD_NRF52840DK
```

```suggestion
config BOARD_PLUGA
```

also in many other places

---

Is there a typo? Isn't this board called `Pulga` not `pLUga`?

---

Yep, the correct name is `pulga`, it means "flea" in Portuguese.